tests/src/FunctionalJavascript/ListPageRssControllerTest.php

```diff
@@ -135,11 +140,11 @@ public function testListPageRssPage(): void {
     $rss_link_selector = 'div.field--name-extra-field-oe-list-page-rss-linknodeoe-list-page a';
     $rss_link = $page->find('css', $rss_link_selector);
     $url = $rss_link->getAttribute('href');
-    $this->assertEqual($url, Url::fromRoute('entity.node.list_page_rss', ['node' => $node->id()])->toString());
+    $this->assertEquals($url, Url::fromRoute('entity.node.list_page_rss', ['node' => $node->id()])->toString());
     $this->drupalGet(Url::fromRoute('entity.node.canonical', ['node' => $node->id()], ['query' => ['random_arg' => 'value']]));
     $rss_link = $page->find('css', $rss_link_selector);
     $url = $rss_link->getAttribute('href');
-    $this->assertEqual($url, Url::fromRoute('entity.node.list_page_rss', ['node' => $node->id()], ['query' => ['random_arg' => 'value']])->toString());
+    $this->assertEquals($url, Url::fromRoute('entity.node.list_page_rss', ['node' => $node->id()], ['query' => ['random_arg' => 'value']])->toString());
```

Comment:
Looks like also you have to swap the order of arguments (see the signature difference of method).

Reply:
This will be fixed on next update.